docs/source/dev_guide/glossary.rst

```diff
@@ -13,6 +13,24 @@ will be used in the following of this guide.
 
 .. contents::
 
+Set up ecpy in developper mode
+------------------------------
```

Comment:
Why the funny line formatting ? Could just make them match 79 characters long ?

Reply:
ok